File: mvc-legacy-server/src/main/java/com/yosep/virtual_thread_vs_webflux_bench_mark_main/db/entity/TestData.java
package com.yosep.virtual_thread_vs_webflux_bench_mark_main.db.entity;

import jakarta.persistence.*;
import lombok.Builder;

@Entity
@Builder
@Table(name = "test_data")
public class TestData {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "name", nullable = false)
    private String name;

    @Column(name = "value_1", nullable = false)
    private Integer value1;

    public TestData() {
    }

    public TestData(String name, Integer value1) {
        this.name = name;
        this.value1 = value1;
    }

    public Long getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public Integer getValue1() {
        return value1;
    }

    public void updateValue1(Integer value1) {
        this.value1 = value1;
    }
}

File: webflux-server/src/main/kotlin/com/yosep/virtual_thread_vs_webflux_bench_mark_main/db/entity/TestData.kt
package com.yosep.virtual_thread_vs_webflux_bench_mark_main.db.entity

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Column
import org.springframework.data.relational.core.mapping.Table

@Table("test_data")
data class TestData(
    @Id val id: Long?,
    val name: String,
    @Column("value_1") val value1: Int
)
